docs/quickstart-config.yaml

```diff
@@ -32,6 +26,3 @@ modules:
 managedApps:
   - io.trino.gateway.ha.GatewayManagedApp
   - io.trino.gateway.ha.clustermonitor.ActiveClusterMonitor
-
-logging:
-    type: external
```

Comment:
Can you update installation doc too?
https://github.com/trinodb/trino-gateway/blob/main/docs/installation.md?plain=1#L103-L112

Reply:
Question. How is this different to httpConfig properties that you can set with https://trino.io/docs/current/admin/properties-logging.html properties using `-Dconfig=var/config.properties`?

Reply:
`httpConfig` works the same as setting `-Dconfig`. There was a discussion on Slack about the pros and cons of moving from YAML to properties, mixing YAML and properties, and keeping everything in YAML. We agreed to keep everything in YAML for now.

Since we chose to use `httpConfig`, `-Dconfig` won't work. They are mutually exclusive: https://github.com/airlift/airlift/blob/538e1ff498bca145c5ec4d582692367694a94fc8/bootstrap/src/main/java/io/airlift/bootstrap/Bootstrap.java#L151-L156